collector/exporter/otelarrowexporter/README.md

```diff
@@ -54,7 +86,45 @@ Several helper files are leveraged to provide additional capabilities automatica
 - [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
 - [Queuing, retry and timeout settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
 
-[beta]: https://github.com/open-telemetry/opentelemetry-collector#beta
-[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
-[core]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol
-[stable]: https://github.com/open-telemetry/opentelemetry-collector#stable
+### Arrow-specific Configuration
+
+In the `arrow` configuration block, the following settings enable and
+disable the use of OTel Arrow as opposed to standard OTLP.
+
+- `disabled`: disables use of Arrow, causing the exporter to use standard OTLP
+- `disable_downgrade`: prevents this exporter from using standard OTLP
+
+The following settings determine the resources that the exporter will use:
+
+- `num_streams`: the number of concurrent Arrow streams
+- `max_stream_lifetime`: duration after which streams are recycled
+
+### Experimental Configuration
+
+The exporter supports configuring compression at the Arrow IPC level.
```

Comment:
Hmm what does IPC stand for?

Reply:
Arrow defines a uni-directional stream named Arrow IPC (generically, "inter-process communication"). I've added a link to https://arrow.apache.org/docs/format/Columnar.html#format-ipc